src/controllers/requests.controller.js

```diff
@@ -0,0 +1,40 @@
+import models from '../db/models';
+import sender from '../services/email.service';
+
+const { Request, User } = models;
+/**
+ * This class creates the user controller
+ */
+export default class UserController {
+  /**
+   * @param {object} req The user's signup details
+   * @param {object} res The user's details returned after signup
+   * @returns {object} A signed up user
+   */
+  static async rejectRequest(req, res) {
+    try {
+      const request = await Request.update(
+        {
+          status: 'declined'
+        }, { returning: true, where: { id: req.params.id } }
+      );
+      const requestResult = request[1][0];
+
+      const user = await User.findOne({ where: { id: requestResult.user_id } });
+      const { first_name: firstName, email } = user;
```

Comment:
Not so major, but for consistency, I will suggest we use snake case when handling persisted data.

Reply:
I get your point @O-Osahon But the reason I did that was because the payload I was passing to my sendgrid API does not allow snake case, that's why I had to assign my `first_name` to a camel case `firstName`. I didn't enjoy doing that too, but that was the only way out.

Reply:
Oh. Alright then.